app/views/events/show.html.haml

```diff
@@ -1,14 +1,14 @@
 .container.py-4.py-lg-5
   .row
-    .col-12
-      %h1
+    .col-12.col-lg-9
+      %h1.mb-4
         = @event.name
         %br
         %small #{humanize_date(@event.date_and_time, @event.ends_at, with_time: true)}
-    .col-12.col-md-9
-      %p.lead= sanitize(@event.description)
+    .col-12.col-lg-9
+      .lead= sanitize(@event.description)
```

Comment:
Does this not still need the p tag to ensure it's a paragraph?

Reply:
@KimberleyCook so this is tricky because if you use HTML in the input and it's wrapped in a p tag itself, it outputs a bunch of empty p tags (+ the HTML you put in the input). Maybe make it so that we require the use of HTML in the input?
<img width="1470" alt="Screenshot 2024-01-03 at 2 24 33 pm" src="https://github.com/codebar/planner/assets/5873816/abff7876-3a3b-4764-a52e-f2b3b5fb0ed2">

Reply:
Gotcha, makes sense. I'd say make sure all the description field needs markdown so you've done. I get it now :)